Tella/Scenes/ContentView/ContentView.swift

```diff
@@ -19,17 +20,17 @@ struct ContentView: View {
             
             if appViewState.currentView == .LOCK {
                 WelcomeView()
-                    .environmentObject(LockViewModel(unlockType: .new))
+                    .environmentObject(LockViewModel(unlockType: .new, appModel: appModel))
             }
             
             if appViewState.currentView == .UNLOCK {
                 let passwordType = AuthenticationManager().getPasswordType()
                 passwordType == .tellaPassword ?
                 UnlockPasswordView()
-                    .environmentObject(LockViewModel(unlockType: .new))
+                    .environmentObject(LockViewModel(unlockType: .new, appModel: MainAppModel.stub()))
```

Comment:
Is there any specific reason to app the stub() instead of appModel?
I think passing the appModel directly is also good is there any specific reason to generate a new instance of MainAppModel in both scenario